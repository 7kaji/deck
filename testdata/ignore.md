# First Slide

This slide will be included.

---

# Second Slide - Ignored

<!-- {"ignore": true} -->

This slide will be ignored and not included in the presentation.

---

# Third Slide

This slide will be included again.

---

# Fourth Slide - Also Ignored

<!-- {"ignore": true} -->

Another slide that should be ignored.

Some more content here.

---

# Fifth Slide

Final slide that will be included.