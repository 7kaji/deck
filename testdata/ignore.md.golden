[
  {
    "layout": "",
    "titles": [
      "First Slide"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This slide will be included."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "First Slide"
      ]
    }
  },
  {
    "layout": "",
    "ignore": true,
    "titles": [
      "Second Slide - Ignored"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This slide will be ignored and not included in the presentation."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Second Slide - Ignored"
      ]
    }
  },
  {
    "layout": "",
    "titles": [
      "Third Slide"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This slide will be included again."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Third Slide"
      ]
    }
  },
  {
    "layout": "",
    "ignore": true,
    "titles": [
      "Fourth Slide - Also Ignored"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Another slide that should be ignored."
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Some more content here."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Fourth Slide - Also Ignored"
      ]
    }
  },
  {
    "layout": "",
    "titles": [
      "Fifth Slide"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Final slide that will be included."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Fifth Slide"
      ]
    }
  }
]